gatereader/reader.py

```diff
@@ -0,0 +1,187 @@
+import serial
+import struct
+from enum import Enum
+from .utils.structparse import mystruct, t, STRUCT_FORMAT
+
+class PacketId(Enum):
```

Comment:
Is "ID" the standard terminology? Type or something would be a lot clearer to me.